ehr/src/org/labkey/ehr/table/DefaultEHRCustomizer.java

```diff
@@ -824,7 +827,12 @@ public void renderGridCellContents(RenderContext ctx, Writer out) throws IOExcep
                         Date date = (Date) ctx.get("date");
                         Object id = ctx.get(ID_COL);
 
-                        out.write("<span style=\"white-space:nowrap\"><a href=\"javascript:void(0);\" onclick=\"EHR.window.ClinicalHistoryWindow.showClinicalHistory('" + objectid + "', '" + id + "', '" + date + "', this);\">[Show Hx]</a></span>");
+                        out.write("<span style=\"white-space:nowrap\"><a class=\"labkey-text-link anm-history\" data-objectid=" + PageFlowUtil.filter(objectid) + " data-id=" + PageFlowUtil.filter(id) + " data-date=" + PageFlowUtil.filter(date) + ">[Show Hx]</a></span>");
```

Comment:
Maybe these values will never have spaces, but they'd break the HTML if they do.
```suggestion
                        out.write("<span style=\"white-space:nowrap\"><a class=\"labkey-text-link anm-history\" data-objectid=\"" + PageFlowUtil.filter(objectid) + "\" data-id=\"" + PageFlowUtil.filter(id) + "\" data-date=\"" + PageFlowUtil.filter(date) + "\">[Show Hx]</a></span>");
```